config.go

```diff
@@ -172,6 +173,13 @@ type RemoteDaemonConfig struct {
 	// MacaroonDir is the directory that contains all the macaroon files
 	// required for the remote connection.
 	MacaroonDir string `long:"macaroondir" description:"The directory containing all lnd macaroons to use for the remote connection."`
+	
+	// CustomMacaroonPath is the path to the custom macaroon that should be
+	// used instead of needing to specify the macaroon directory that
+	// contains all of lnd's macaroons. The custom macaroon MUST have all
+	// permissions that all the subservers use, otherwise permission errors
+	// will occur.
+	CustomMacaroonPath string `long:"custommacaroonpath" description:"The path to the custom macaroon to use. Cannot be specified at the same time as macaroondir. The custom macaroon must contain ALL permissions required for all subservers to work, otherwise permission errors will occur."`
```

Comment:
I like that we can now just specify the admin macaroon file instead of the full dir. I wonder if we should just deprecate `macaroondir` and change `custommacaroonpath` to `macaroonpath` instead of keeping both flags permanently. I believe that having `custommacaroonpath` adds a bit of confusion. My hunch is that everyone should just use this flag pointed at the admin macaroon. Maybe I'm misunderstanding the intention here though.

Reply:
Great point. I've added a deprecation comment to the dir flag and renamed the new one to `macaroonpath`.

Reply:
Awesome! 🥇 

Not sure if you want to do it in this PR, but we should also update the docs & examples to use `macaroonpath` as well.

Reply:
Yes, I did update the docs. Or did I overlook anything?

Reply:
Oh, my bad. I missed that on my end. Looks good 👍